lib/tasks/green_lanes.rake

```diff
@@ -22,4 +22,34 @@ namespace :green_lanes do
     Rails.root.join('data/green_lanes/categories.json').write \
       JSON.pretty_generate(@json)
   end
+
+  desc 'Import Themes data'
+  task import_themes: :environment do
+    raise 'Not in XI environment' unless TradeTariffBackend.xi?
+
+    source_file = Rails.root.join('data/green_lanes/themes.csv')
+    raise "Cannot read file '#{source_file}'" unless File.file?(source_file)
+
+    @existing_themes = GreenLanes::Theme.all.index_by { |t| [t.section, t.subsection] }
+
+    GreenLanes::Theme.db.transaction do
+      CSV.foreach(source_file, headers: true) do |row|
+        section, theme = row['Theme'].split(' ', 2)
+        theme = theme.strip
+        section = section.strip.gsub(/^(\d+\.\d+)\./, '\1')
+        section, subsection = section.split('.').map(&:to_i)
+
+        instance = @existing_themes[[section, subsection]]
+        instance ||= GreenLanes::Theme.new(section:, subsection:)
+        instance.theme = theme
+        instance.description = row['Full description']
+        instance.category = row['Category implied']
+        if instance.valid?
```

Comment:
instead of checking `valid?`,  we can just: 
`save(raise_on_failure: true)`

without even rescuing the error,
but in case you want:

```
rescue Sequel::ValidationFailed => e
  raise "Import error! #{instance.inspect}"
  ```

Reply:
good suggestion!